pods/initContainers/volume-init-containers.yaml

```diff
@@ -18,9 +18,10 @@ metadata:
     ]'
 spec:
   containers:
-    - image: "docker.io/deshuai/hello-pod:latest"
+    - image: "docker.io/ocpqe/hello-pod:latest"
       imagePullPolicy: IfNotPresent
       name: hello-pod
+      command: ["/bin/sh", "-c", "while true; do cat /tmp/hello.txt; sleep 5; done"]
```

Comment:
suggust use yaml style rather than json style here.
      command:
      - /bin/sh
      - -c
      - while true; do cat /tmp/hello.txt; sleep 5; done